### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by sfod**
- Add missing documentation for operations.
- Add support for multiline documentation.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: greengrass-ipc-model/main.smithy [hunk 1]
```diff
@@ -192,21 +193,22 @@ operation SubscribeToIoTCore {
...
-// Retrieves a secret stored in AWS secrets manager
+/// Retrieves a secret stored in AWS secrets manager
 operation GetSecretValue {
     input: GetSecretValueRequest,
     output: GetSecretValueResponse,
     errors: [UnauthorizedError, ResourceNotFoundError, ServiceError]
 }
 
-// Generate a password for the HttpDebugView component
+/// Generate a password for the HttpDebugView component
```
Comment (MikeDombo) [inline greengrass-ipc-model/main.smithy:211]: LocalDebugConsole

File: greengrass-ipc-model/main.smithy [hunk 2]
```diff
@@ -242,44 +244,49 @@ operation ListNamedShadowsForThing {
...
+}
 
+/// Send a request to authorize action on some resource
 operation AuthorizeClientDeviceAction {
      input: AuthorizeClientDeviceActionRequest,
      output: AuthorizeClientDeviceActionResponse,
      errors: [UnauthorizedError, ServiceError, InvalidArgumentsError, InvalidClientDeviceAuthTokenError]
  }
 
+/// Send component metrics
```
Comment (MikeDombo) [inline greengrass-ipc-model/main.smithy:289]: Only usable by AWS components

File: greengrass-ipc-model/main.smithy (hunk 1 above)
Comment (sfod) [reply greengrass-ipc-model/main.smithy:211]: To where `LocalDebugConsole ` should be added?

Comment (MikeDombo) [reply greengrass-ipc-model/main.smithy:211]: It isn't called HttpDebugView. It is LocalDebugConsole

**Review by MikeDombo — APPROVED** [review]

**Pull request merged**